exercises_on_code_quality/createAPhoneNumber.js

```diff
@@ -0,0 +1,6 @@
+const createPhoneNumber = (numbers) => {
+  const x = numbers.slice(0, 3).join("");
+  const y = numbers.slice(3, 6).join("");
+  const z = numbers.slice(6).join("");
```

Comment:
I would suggest you to not create these variables since they are not being reused.
Instead you can add those statements in return